HW_1/task2.py

```diff
@@ -0,0 +1,13 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0)
+
+n = int(input("Введите трехзначное число: "))
+ 
+x1 = n % 10
+n = n // 10
+x2 = n % 10
+n = n // 10
+result = n + x1 + x2
```

Comment:
Можно было не использовать промежуточные переменные.
```suggestion
x1 = n % 10
x2 = n // 10 % 10
n = n // 100
result = n + x1 + x2
```